packages/client/src/rx/operators/tap.ts

```diff
@@ -0,0 +1,26 @@
+import { MonoTypeOperatorFunction, Observer } from '../types';
+
+export function tap<TValue, TError>(
```

Comment:
Not sure about the name of this, but I don't have a better one atm (other than tee).

Reply:
The naming here is from https://rxjs.dev/api/operators/tap